<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite failures="1" name="cucumber.runtime.formatter.JUnitFormatter" skipped="0" tests="1" time="28.555168">
<testcase classname="To validate login functionality in Facebook application" name="To validate login functionality with invalid username and password" time="28.555168">
<failure message="java.lang.AssertionError&#13;&#10;&#9;at org.junit.Assert.fail(Assert.java:86)&#13;&#10;&#9;at org.junit.Assert.assertTrue(Assert.java:41)&#13;&#10;&#9;at org.junit.Assert.assertTrue(Assert.java:52)&#13;&#10;&#9;at org.stepdefinition.LoginSteps.the_user_has_to_clcik_the_login_button(LoginSteps.java:31)&#13;&#10;&#9;at ✽.The user has to  clcik the login button(file:src/test/resources/Features/login.feature:9)&#13;&#10;"><![CDATA[Given The user should be in Facebook page...................................passed
When The user has to fill "mano"  and "5678"................................passed
And The user has to  clcik the login button.................................failed
Then The user should be in invalid page.....................................skipped

StackTrace:
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.stepdefinition.LoginSteps.the_user_has_to_clcik_the_login_button(LoginSteps.java:31)
	at ✽.The user has to  clcik the login button(file:src/test/resources/Features/login.feature:9)
]]></failure>
</testcase>
</testsuite>
